compose.production.yaml

```diff
@@ -7,11 +7,12 @@
 
 version: "3.8"
 services:
-  web:
-    profiles: ["ol-web1", "ol-web2"]
+  # Use YAML anchor (&) and alias (*) to provide different profiles and hostnames
+  web:  &web_configuration
```

Comment:
Note compose files inherit from each other (and this `web`  section inherits from `compose.yaml`). Does inheriting work correctly? So when we create web2 does web2 have all the inherited properties? For example, the `command` field is inherited.

Try adding `web2` to `compose.override.yaml`. That'll let you test this configuration out locally.

Reply:
Does `test_docker_web_nodes()` not cover these conditions?

Reply:
That tests just the yaml we need to confirm this actually runs correctly when using docker. The python test only checks the fields `profile` and `hostname`. We need to confirm that `command`, which is defined in compose.yaml, exists.

Either way, testing this locally first will give us a lot of clarity/confidence!

Reply:
> The python test only checks the fields `profile` and `hostname`.

All keys and values are tested...
`assert web == web2  # All remaining keys and values should be the same.`

I take a look at `command`